src/core/ActiveStatus.cpp

```diff
@@ -690,8 +732,8 @@ void ActiveStatus::catchUpManifestLogs() {
                 LOG(INFO, "[ActiveStatus]          |"
                         "Replay assignBlock log record with %lu blocks.", blocks.size());
                 assert(header.numBlocks == 1);
-                mBlockArray.push_back(blocks[0]);
-                mEof = header.opaque.extendBlock.eof;
+//                mBlockArray.push_back(blocks[0]);
```

Comment:
Wrong, you should skip the log size when writing new log. but not fix in catchUpLogic.
A concurrent read might need this log